### PR TITLE
Fix bigquery config for serving store

### PR DESCRIPTION
**Pull request by zhilingc**
**What this PR does / why we need it**:
Running feast serving with bigquery as a backend throws the following exception:
```
Failed to instantiate [feast.serving.specs.CachedSpecService]: Factory method 'specService' threw exception; nested exception is com.google.protobuf.InvalidProtocolBufferException: Cannot find field: staging-location in message feast.core.Store.BigQueryConfig
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:584)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:90)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:370)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1336)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:572)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:495)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:317)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:315)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:762)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:881)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:551)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:142)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:386)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:307)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1242)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1230)
	at feast.serving.ServingApplication.main(ServingApplication.java:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51)
```

This is because the application.yml configuration keys do not match the proto field names. This PR fixes that.

### CODE REVIEW

File: serving/src/main/resources/application.yml [hunk 1]
```diff
@@ -36,11 +36,11 @@ feast:
         # to download the batch features.
         # For example: gs://mybucket/myprefix
         # Please omit the trailing slash in the URI.
-        staging-location: gs://mybucket/myprefix
+        staging_location: gs://mybucket/myprefix
         # Retry options for BigQuery retrieval jobs
-        bigquery-initial-retry-delay-secs: 1
+        bigquery_initial_retry_delay_secs: 1
```
Comment (woop) [inline serving/src/main/resources/application.yml:41]: Should the `bigquery_` be removed?
https://github.com/gojek/feast/blob/master/protos/feast/core/Store.proto#L124

Comment (woop) [reply serving/src/main/resources/application.yml:41]: and `secs` to `seconds`

Comment (zhilingc) [reply serving/src/main/resources/application.yml:41]: yes, sorry. Just noticed

Comment (zhilingc) [reply serving/src/main/resources/application.yml]: Fixed, could you check?

**Pull request merged**